charts/agent/templates/deployment.yaml

```diff
@@ -21,26 +21,46 @@ spec:
       {{- end }}
         updatedAt: {{ now | quote }}
       labels:
-        app.kubernetes.io/name: {{ include "convoy-worker.name" . }}
+        app.kubernetes.io/name: {{ include "convoy-agent.name" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}
     spec:
       initContainers:
         - name: wait-for-migrate
```

Comment:
I am sorry if I am re-opening this but this line is wrong, we can rename this container or simply remove the line and add "-" to image line.